packages/optimistic-oracle-v2/README.md

```diff
@@ -0,0 +1,7 @@
+# Optimistic Oracle V2 Subgraph
+
+## Deploying the subgraph
+
+- Mainnet: `yarn prepare:mainnet && yarn deploy:mainnet`
+- Kovan: `yarn prepare:kovan && yarn deploy:kovan`
```

Comment:
Kovan can be removed and maybe add the [rest of networks](https://github.com/UMAprotocol/subgraphs/blob/master/packages/optimistic-asserter/README.md)